summit-demo/README.md

```diff
@@ -6,7 +6,9 @@
 
 2. Log in. (use prod credentials, there doesn't seem to be stg.accounts)
 
-3. Create an API token.
+3. Create your personal API token.
```

Comment:
Maybe link https://git.stg.centos.org/settings/token/new.

Worth mentioning the ACLs needed?